hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -367,7 +367,8 @@
 
   private HbckChore hbckChore;
   CatalogJanitor catalogJanitorChore;
-  private DirScanPool cleanerPool;
+  private DirScanPool archiveCleanerPool;
```

Comment:
Better add some comments here? I think the logCleanerPool is for removing wals, and the archiveCleanerPool is for all other cleaners? And do we need DirScanPool for cleaners other than log and hfile? If not, let's just name them logCleanerPool and hfileCleaner pool?